tests/loggen/file_reader.c

```diff
@@ -174,8 +304,7 @@ read_next_message_from_file(char *buf, int buflen, int syslog_proto, int thread_
   static int lineno = 0;
   int linelen;
 
-  char host[128], program[128], pid[16];
-  char *msg = NULL;
+  SyslogMsgElements parsed_elements;
```

Comment:
This should be memset to zero, because a few lines below we check
parsed_elements.msgid[0] which might be uninitialized.

Reply:
done